index.rst

```diff
@@ -1,58 +1,129 @@
-  ^^^^^^^^^^^^^^
+   Its purpose is to document the transition so as to allow Ops-funded developers to continue to maintain and evolve the service, and to allow the service's deployment costs to be covered by Ops infrastructure or funding. It can also be used to describe responsibilities for the service when not a direct continuation of the model used in construction.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+   The process is complete when the relevant AD(s) and/or Director reviews the note and the note is published (merged to master).
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+.. Add content here.
 
-     Caption text.
+What is covered
+===============
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+This delivery note covers the Discourse-based web forum service deployed as community.lsst.org
 
-   Feel free to delete this instructional comment.
+The web forum is a destination for long-lived (compared to Slack) discussion forum replacing mailing lists for internal project use, as well as a community-facing forum designed to involve the community in discussions with the project as well as their peers in an asynchronous and searchable manner.
 
-:tocdepth: 1
+What artifacts (code, docs, infrastructure) are involved?
+---------------------------------------------------------
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+The following repositories include visual (theme) styling components:
 
-.. sectnum::
+- https://github.com/lsst-sqre/discourse-rubin-theme
+- https://github.com/lsst-sqre/discourse-alt-lock-icon
+- https://github.com/lsst-sqre/discourse-rubin-header
+- https://github.com/lsst-sqre/discourse-rubin-footer
 
-.. TODO: Delete the note below before merging new content to the master branch.
+The following repo documents an example of how to use the Discourse API:
 
-.. note::
+- https://github.com/lsst-sqre/community_api
 
-   **This technote is not yet published.**
+Additionally the following resources are involved in the operation of this service:
+
+- DigitalOcean droplet
+- Mailchimp account for email
+- Let's Encrypt account for TLS
+- AWS S3 bucket for backups
+- Slack application to bridge Discourse and Slack
+- Twitter app for OAuth login
+- GitHub OAuth app for login
+- 1Password for secrets management
+
+The motivation for the service's creation and outline of its elements can be found at https://sqr-011.lsst.io/#community-forum-and-mailing-lists
+
+What costs does the service incur?
+-----------------------------------
+
+The service is hosted on a Digital Ocean droplet and costs approximately $30/month billed directly to an AURA Purchasing credit card under a blanket PO against CS008.1.02C.10.01.KLM21001A
```

Comment:
Suggest "The service is hosted on a Digital Ocean droplet and costs approximately $30/month billed directly to an AURA Purchasing credit card under a blanket PO against CS008.1.02C.10.01.KLM21001A. A new PO will be issued for Operations under CS002."

Reply:
Thanks, the billing for this is on the same invoice as the mailing list server (lists.lsst.org) but I will (1) turn around a quick delivery note for that (2) it's only $10/month